tests/retrieveBearerToken.test.js

```diff
@@ -0,0 +1,109 @@
+import { createMocks } from "node-mocks-http";
+import client from "next-auth/client";
+import { retrieve } from "../pages/api/id/[form]/bearer";
+import fetchMock from "jest-fetch-mock";
+
+jest.mock("next-auth/client");
+
+describe("Test bearer token retrieve api endpoint", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    fetchMock.enableMocks();
+  });
+
+  it("Shouldn't allow GET request without a valid form's id", async () => {
+    const mockSession = {
+      expires: "1",
+      user: { email: "admin2@cds.ca", name: "Admin2 user", image: "null" },
+    };
+
+    client.getSession.mockReturnValueOnce(mockSession);
+    fetchMock.mockResponseOnce(JSON.stringify({ token: "akalkdfljasdlkfdsljasfdskfj" }));
```

Comment:
Not entirely sure what this is doing. I see it in some other existing test files although I don't see why fetch is being mocked here